service-seed-without-router/all-actors/src/main/java/org/sunbird/HealthActor.java

```diff
@@ -4,17 +4,35 @@
 import org.sunbird.request.Request;
 import org.sunbird.response.Response;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
 @ActorConfig(
         tasks = {"health"},
         dispatcher = "",
         asyncTasks = {}
 )
 public class HealthActor extends BaseActor {
+
     @Override
     public void onReceive(Request request) throws Throwable {
-        Response response = new Response();
-        response.put("Response",request.getRequest());
-        response.put("healthy",true);
-        sender().tell(response,self());
+        try {
+            String uuid = UUID.randomUUID().toString();
+            Map<String, Object> mdc;
+            mdc = new HashMap<>();
+            mdc.put("requestId",uuid );
```

Comment:
Not sure if this is required since BaseActor is already set this.

Reply:
done